CondCore/ESSources/test/stubs/PedestalsAnalyzer.cc

```diff
@@ -9,67 +9,60 @@ Toy EDProducers and EDProducts for testing purposes only.
 #include <string>
 #include <iostream>
 #include <map>
-#include "FWCore/Framework/interface/EDAnalyzer.h"
-#include "FWCore/Framework/interface/Event.h"
-#include "FWCore/Framework/interface/ESHandle.h"
-#include "FWCore/Framework/interface/MakerMacros.h"
-
-#include "FWCore/Framework/interface/EventSetup.h"
 
 #include "CondFormats/Calibration/interface/Pedestals.h"
 #include "CondFormats/DataRecord/interface/PedestalsRcd.h"
 
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
 #include "TFile.h"
 
 using namespace std;
 
 namespace edmtest {
-  class PedestalsAnalyzer : public edm::EDAnalyzer {
+  class PedestalsAnalyzer : public edm::one::EDAnalyzer<> {
```

Comment:
```suggestion
  class PedestalsAnalyzer : public edm::one::EDAnalyzer<edm::one::WatchRuns> {
```
this needs to watch runs, as it has a `beginRun` method (that should be marked as `override`)